### DISCUSSION TITLE
Basic questions around this project

### DISCUSSION THREAD

**Discussion by brainstormi** [nliaudat/esp32_8ch_motor_shield#40] [General]
Hi...
    First of all, apologize me if my questions are too basic, I'm quite weak at electronics and heating floor topics. I hope you can support me for having a better understanding of this solution.
    1. I have some difficulties to find DC valve actuators that match the voltage requirements. Basically all I've found are 230v AC actuators or 24V DC ones. Market is full of them, mostly the first ones, but suitable for this project I was only able to found HmIP-VDMOT suggested here and not very good comments about the quality or robustness of them. What are other options in the market?.
    2. In the wiring diagram for HmIP-VDMOT and new PCB 2 threads connectors, what are the two wires connected? 3.3v and GND?
    3. Most systems use 230V AC actuators and if I understand it correctly these behave as ON/OFF connectors not giving the possibility to adjust it proportionally. Taking into account the big inertia of these type of floors, more if they're low temperature water based, in your experience is it really worth it to make this kind of fine adjustments reducing the life of the actuator themselves? It seems that big brands like Uponor also bet for 230v actuators. what am i missing?.
    4. In the new AC power in connector of 2 two threads, is it needed to solder ground cable? where?.
Regards,

**Comment by nliaudat** [comment]
Hi, 
1) AC 230 and DC 24 are on/off valves. You can control them with a simple relay board.
You can find proportionnals valves like Equiva Model N or https://github.com/nliaudat/floor-heating-proportional-valve without the controller board
2) Yes, the refer to https://github.com/nliaudat/esp32_8ch_motor_shield/blob/main/VDMOT_wiring.md
3) This is a min movement (5% per default) change cursor you can define. My valves runs without problem since 3 years. 
4) The AC ground in not connected, but you can solder it on DC GND holes if you want. There is isolation from AC/DC

**Reply by brainstormi** [reply]
Thanks for your response, very useful!
What about your feedback about the use of proportional valves instead standard AC on/off mechanism for heated water floor. What are the benefits of this approach in your installation?.

**Reply by nliaudat** [reply]
With proportionnal valves, you have no more floor heating inertia.

**Reply by brainstormi** [reply]
Maybe this is the my most dummy question :) But floor heating inertia is not one the biggest benefits of this type of systems?, could retain temperature for more time without wasting energy compared to heat pump systems for example, is this last system when you shutdown it the room drops its temperature a lot quicker.

**Reply by nliaudat** [reply]
The right comment should be `"With proportionnal valves, you **limit effects of** floor heating inertia."`

 The low temperature floor heating has inertia and it could be a good or bad thing. Here are 2 examples of bad inertia :

- The initial load (start) on low temp (28°) floor heating take up to 2 days to heat your room
- If the external temperature rise, the same rules applies, and you can have overheating. 

With proportionnal valves, the only advantage over traditionnal 230V actuators is that the water flow is "_always on_", but with exactly the good limited flow.

The time to open or close traditionnal actuators take up to 45min cause they need to heat the inside vax to expand.
The result is a mean between ON/OFF signal by termostat. ( with bimetal thermostat, that's worse)

With a controller and proportionnals valves, you can setup weather forecast, away home, etc and you have more temperature sensors.
_(In my case, I've setup the temperature sensors at ~1.5m height, but you can have about 2° of temp difference at 0.5m. You can setup the precise temperature you like lying on your couch)_

For advantages of proportional valve under floor heating, ChatGPT answers : 

```
Proportional valves offer several advantages in underfloor heating systems. Here are some of the key benefits:


    Precise Temperature Control: Proportional valves allow for accurate and continuous control of the flow rate of hot water in underfloor heating systems. They can modulate the flow based on the desired temperature, ensuring that the floor maintains a consistent and comfortable temperature throughout the space. This level of control is particularly important in underfloor heating, where maintaining even heat distribution is crucial.

    Energy Efficiency: By modulating the flow rate, proportional valves can optimize energy consumption in underfloor heating systems. They help prevent overheating and unnecessary energy loss by adjusting the flow to match the actual heating requirements of the room. This results in improved energy efficiency and reduced operating costs over time.

    Comfort and Zoning: Underfloor heating systems often utilize zoning, where different areas or rooms have separate temperature controls. Proportional valves allow for independent temperature control in each zone, ensuring personalized comfort levels. Each zone can have its own valve that modulates the flow of hot water based on the specific heating demands of that area, providing individualized comfort for the occupants.

    Faster Response Time: Proportional valves can react quickly to changes in temperature demands. They can adjust the flow rate rapidly, allowing the underfloor heating system to respond promptly to temperature variations in the room. This faster response time helps maintain a stable and comfortable indoor environment, reducing temperature fluctuations and improving overall comfort.

    System Protection: Proportional valves often include features to protect the underfloor heating system. For example, some valves have built-in temperature sensors that monitor the water temperature and can shut off the flow if it exceeds a certain limit, preventing damage to the system. This safeguards the components and prolongs the lifespan of the underfloor heating system.

Overall, proportional valves offer precise control, energy efficiency, comfort customization, fast response times, and system protection advantages for underfloor heating systems. They enhance the performance and reliability of the system, ensuring optimal comfort and efficiency for the occupants.
```

**Reply by brainstormi** [reply]
Thank you very much for your detailed response, I'm convinced. It's the time to start tinkering with the electronics :)
